src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADDBOOK = 'ADDBOOK';
+const REMOVEBOOK = 'REMOVEBOOK';
```

Comment:
- [ ] [OPTIONAL] It will be better to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so make it like `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`